README.md

```diff
@@ -1,7 +1,52 @@
+[ ![Codeship Status for
+CatchRelease/arbor](https://app.codeship.com/projects/0fd77eb0-aede-0136-de71-6acd4e02bc2c/status?branch=master)](https://app.codeship.com/projects/310054)
+[![code style:
+prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier)
+
 # Arbor
 
-React component library for Catch&Release
+Arbor is a set of React UI components for [Catch&Release's](https://www.catchandrelease.com/) design system.
+It is built using [emotion](https://emotion.sh/) and [styled-system](https://github.com/jxnblk/styled-system).
+
+## Getting Started
+
+### Installation
+
+Install the [Arbor package](https://www.npmjs.com/package/@catchandrelease/arbor) and its dependencies.
+
+```
+npm install --save @catchandrelease/arbor emotion emotion-theming react-emotion react react-dom
```

Comment:
Is there a reason to add the dependencies even if they are listed in the package.json and will be installed automatically?

Reply:
Good question.

To be honest, i don't think so. I borrowed this from a couple of README's that I looked at. I'll remove it in favor of just `npm install --save @catchandrelease/arbor`